### PR TITLE
Fixes for #891

### PR DESCRIPTION
**Pull request by nbeliy**
Added instructions to avoid Tcl's couldn't connect to display error to installation instructions.

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -33,6 +33,21 @@ github using ``pip`` on a Python 3 environment: ::
...
+	MRIQC uses matplotlib to create graphics. By default matlpotlib 
+	uses interactive Tck interface, which requires graphic enveronment working. 
+	If such enveronment is not present (for ex. when running under tmux),
+	you may see an error: ::
+	
+		_tkinter.TclError: couldn't connect to display "localhost:10.0"
+	
+	To fix it you need uncomment "backend : Agg" line in configuration file 
+	of matplotlib. The location of configuration file can be retrieved
+	from python command prompt: ::
```
Comment (oesteban) [inline docs/source/install.rst:47]:
```suggestion
	There are two pathways to fix this issue.
	One is setting up a virtual display with a tool like XVfb.
	Alternatively, you can configure your matplotlib distribution to perform on
	head-less mode by default.
	That is achieved by uncommenting the ``backend : Agg`` line in the matplotlib's
	configuration file.
	The location of the configuration file can be retrieved with Python::
```

File: docs/source/install.rst [hunk 2]
```diff
@@ -33,6 +33,21 @@ github using ``pip`` on a Python 3 environment: ::
...
+	
+	> import matplotlib
+	> print(matplotlib.matplotlib_fname())
```
Comment (oesteban) [inline docs/source/install.rst:50]:
```suggestion
	  >>> import matplotlib
	  >>> print(matplotlib.matplotlib_fname())

      Alternatively, you can issue the following shell command-line to edit this setting::
      
        $ sed -i 's/\(backend *: \).*$/\1Agg/g' $( python -c "import matplotlib; print(matplotlib.matplotlib_fname())" )

```

**Review by oesteban — COMMENTED** [review]
LGTM after some suggestions and typos are fixed

File: docs/source/install.rst (hunk 2 above)
Comment (nbeliy) [reply docs/source/install.rst:50]: Not sure that `sed` is needed for changing one value in one configuration file. 
Also a typo in line may lead to unpredictable results.

File: docs/source/install.rst [hunk 3]
```diff
@@ -33,6 +33,21 @@ github using ``pip`` on a Python 3 environment: ::
 .. warning::
 
 	As of MRIQC 0.9.4, Python 2 is no longer supported.
+	
+matplotlib/Tcl warning::
```
[the remark this answers is not in the document]
Comment (nbeliy) [reply docs/source/install.rst:37]: This is a warning concerning Tcl, I think making it explicit in title may be useful.

[the remark this answers is not in the document]
Comment (oesteban) [reply docs/source/install.rst:37]: Is your proposal RST valid? (I don't think so)

[the remark this answers is not in the document]
Comment (nbeliy) [reply docs/source/install.rst:37]: No idea, I don't know RST.

File: docs/source/install.rst (hunk 2 above)
Comment (oesteban) [reply docs/source/install.rst:54]: Sure, that's why we are still providing both mechanisms for people to do this. In general, a typo on any command line could potentially lead to unpredictable results - I think we can trust users know that.

File: docs/source/install.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (oesteban) [reply docs/source/install.rst:37]: Hence, my suggestion. The goal is that this document renders correctly. If you don't accept this suggestion, it won't render correctly.

File: docs/source/install.rst (hunk 2 above)
Comment (oesteban) [reply docs/source/install.rst:54]: Also, if you are using tmux or a headless terminal (e.g., on a cluster) that means you are probably familiar with some shell rudiments. And if you are not savvy on shell, I think copying and pasting this is much easier than running python and then getting trapped into `vim` or `emacs` or `nano`.

IMHO this is safe to suggest, and I'd be actually inclined to make the suggestion the default.

File: docs/source/install.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (oesteban) [reply docs/source/install.rst:37]: Yes, not rendering properly: https://3541-49015296-gh.circle-artifacts.com/0/docs/_build/html/install.html

File: docs/source/install.rst (hunk 2 above)
Comment (nbeliy) [reply docs/source/install.rst:55]: You are the boss

**Comment by oesteban** [comment]
Cool! It's looking good - https://3555-49015296-gh.circle-artifacts.com/0/docs/_build/html/install.html

One final request: could you make sure to convert tabs into spaces? The weird indentation of the initial block is caused by that, I believe.

**Comment by nbeliy** [comment]
> One final request: could you make sure to convert tabs into spaces? The weird indentation of the initial block is caused by that, I believe.

Done, hope I didn't forgot any tab

**Pull request merged**